lib/y2_installbase.pm

```diff
@@ -98,10 +98,10 @@ sub deal_with_dependency_issues {
         send_key_until_needlematch 'packages-section-selected', 'tab';
         send_key 'ret';
     }
-    if (get_var("WORKAROUND_DEPS")) {
+    if (check_var("WORKAROUND_DEPS", '1')) {
         y2_logs_helper::workaround_dependency_issues;
     }
-    elsif (get_var("BREAK_DEPS")) {
+    elsif (check_var("BREAK_DEPS", '1')) {
         y2_logs_helper::break_dependency;
```

Comment:
also for the BREAK_DEPS
 grep -risn BREAK_DEPS *
lib/y2_installbase.pm:104:    elsif (get_var("BREAK_DEPS")) {
tests/autoyast/installation.pm:128:    push(@needles, 'manual-intervention') if get_var("BREAK_DEPS");
tests/installation/change_desktop.pm:38:    if (check_screen('dependency-issue', 0) && get_var("BREAK_DEPS")) {